src/pages/iou/IOUModal.js

```diff
@@ -92,6 +92,13 @@ const defaultProps = {
     currentUserPersonalDetails: {
         localCurrencyCode: CONST.CURRENCY.USD,
     },
+    personalDetails: {},
+    iou: {
+        creatingIOUTransaction: false,
+        error: false,
+        isRetrievingCurrency: false,
```

Comment:
`isRetrievingCurrency` is removed here https://github.com/Expensify/App/pull/4225, so we can remove it too from the prop types.

Reply:
Yeah, nice catch!